examples/kafka-hub/docker-compose.yaml

```diff
@@ -87,7 +87,7 @@ services:
             ZOOKEEPER_TICK_TIME: 2000
 
     broker:
-        image: 'confluentinc/cp-server:latest'
+        image: 'ayeshalmeida/cp-server:7.5.0'
```

Comment:
Using images from personal docker repositories are fine?

Reply:
Yes, this is for a demo purposes only.